starport/pkg/xcobra/flags.go

```diff
@@ -0,0 +1,44 @@
+package xcobra
+
+import (
+	"github.com/spf13/pflag"
+)
+
+// FlagValueEmptiness represent empty/filled flag value statuses.
+type FlagValueEmptiness int
+
+const (
+	// FlagsAllEmpty indicates that all compared flag values are empty.
+	FlagsAllEmpty FlagValueEmptiness = iota
+
+	// FlagsSomeEmpty indicates that some compared flag values are empty.
+	FlagsSomeEmpty
+
+	// FlagsAllFilled indicates that all compared flag values are filled.
+	FlagsAllFilled
+)
+
+// CheckFlagValues checks the emptiness of given flags with names.
+func CheckFlagValues(flags *pflag.FlagSet, names []string) (emptiness FlagValueEmptiness, err error) {
```

Comment:
I would add a test for the 3 cases

Reply:
solved by 941f52fc3071a645f3009d3e5c768b6f0bd28603.